community-operators/tidb-operator/tidb-operator.1.0.0-beta1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,214 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: tidb-operator.1.0.0-beta1
+  namespace: placeholder
+  annotations:
+    alm-examples: |-
+        [  
+          {
+            "apiVersion": "pingcap.com/v1alpha1",
+            "kind": "TidbCluster",
+            "metadata": {
+              "name": "demo",
+              "namespace": "tidb1"
+              },
+            "spec": {
+              "pd": {
+                  "image": "pingcap/pd:v2.1.0",
+                  "imagePullPolicy": "IfNotPresent",
+                  "limits": {},
+                  "nodeSelectorRequired": true,
+                  "replicas": 3,
+                  "requests": {
+                    "storage": "5Gi"
+                  },
+                  "storageClassName": "pd-ssd"
+                },
+              "pvReclaimPolicy": "Retain",
+              "schedulerName": "tidb-scheduler",
+              "services": [
+                "name": "pd",
```

Comment:
From our tests, this should be a list of objects, like so:
```
"services": [
  {
    "name": "pd",
    "type": "ClusterIP
  }
],
```
Please let us know if something about this seems off. We aren't experts on the operator, but this is what we could run without errors.

Reply:
Made the change and CI passed with slightly improved scorecard: https://travis-ci.com/operator-framework/community-operators/builds/108997561

Is the goal to reach 100% on the scorecard? Should we make additional changes based on the 'SUGGESTION'?

Reply:
No need for 100% on the scorecard. You can make the additional changes as suggested, but that can be done in a later release. Mostly it's important that the operator is able to respond to the example CR when installed via OLM, and I am able to gauge that through the scorecard test by checking if the `status` section of the CR is present, since this is usually added by operators.